AlphaWallet/RPC/Commands/web3swift-pod/GetInterfaceSupported165Encode.swift

```diff
@@ -0,0 +1,7 @@
+// Copyright © 2019 Stormbird PTE. LTD.
+import Foundation
+
+struct GetInterfaceSupported165Encode {
```

Comment:
`GetInterfaceSupported165Encode` and `GetIsERC721` looks identical. Maybe rename the latter to the former? The new coordinator still looks necessary though.